aee.js

```diff
@@ -5,3 +5,17 @@ Array.prototype.each = function (callback) {
   }
   return this
 }
+
+Array.prototype.any = function (spec) {
+  const len = this.length
+  if (typeof spec !== 'function') {
+    searchedValue = spec
+    spec = function (element, index) {
```

Comment:
You can omit the parameters that you are not planning to use. I mean, while you don't change the position of the ones you need to use.